tools/topology/sof/pipe-codec-adapter-playback.m4

```diff
@@ -0,0 +1,112 @@
+# Low Latency Passthrough Pipeline and PCM
+#
```

Comment:
@johnylin76 I don't understand exactly WHY this change is needed? 

pipe-processing-playback.m4 file has the exact purpose of a common codec adapter playback pipeline topology that others can use in their m4 files to create a topology file.

Reply:
this does feel very similar to pipe-processing, @johnylin76 can you share the intent of this change?

Reply:
The intent of this change is to create a template of pipi-playback m4 for codec adapter codecs. For more and more codecs in the future, it can minimize the effort and avoid errors on copy/paste.
Here is another codec we are developing: https://github.com/thesofproject/sof/pull/3718/commits/f960b11ffbd4ecb0c360bf1ac6978b88788ee23d#diff-e5866322398b5edd5d9432288f69de57626645f898caf11d02e3923787dd7eaa

Reply:
Isnt' that what https://github.com/thesofproject/sof/blob/master/tools/topology/sof/pipe-processing-playback.m4 is though? Sorry still unclear on this change.

Reply:
Could a developer create a pipe-xxx-playback.m4 for a new codec by including pipe-processing-playback.m4 and then re-declare its control bytes of setup config and runtime param? or change the byte control port name? I don't know how to do it but it can be achieved then this change is not necessary.

Reply:
I see this is useful change, as a higher level abstraction on the codec adapter. Fwiw, this could be one of the last complex topologies using M4 as the topology 2.0 work is maturing.

Reply:
Ah, yes, hopefully tplg2 will have a useful mechanism to pass in byte, and control sizes